tests/integration/kubernetes/k8s-cpu-ns.bats

```diff
@@ -100,6 +101,7 @@ teardown() {
 	[ "${KATA_HYPERVISOR}" == "firecracker" ] && skip "test not working see: ${fc_limitations}"
 	[ "${KATA_HYPERVISOR}" == "fc" ] && skip "test not working see: ${fc_limitations}"
 	[ "${KATA_HYPERVISOR}" == "dragonball" ] && skip "test not working see: ${dragonball_limitations}"
+	[ "${KATA_HYPERVISOR}" == "qemu-runtime-rs" ] && skip "test not working see: ${dragonball_limitations}"
```

Comment:
Why here has `${dragonball_limitations}` ? I am curious about it.  Can this be changed as `[ "${KATA_HYPERVISOR}" == "dragonball" ] || [ "${KATA_HYPERVISOR}" == "qemu-runtime-rs" ] && skip "test not working see: ${dragonball_limitations}"` if both of them use the `"test not working see: ${dragonball_limitations}"` ?

Reply:
Good point @Apokleos

dragonball_limitations is https://github.com/kata-containers/kata-containers/issues/6621 that points to https://github.com/kata-containers/tests/issues/5391 then https://github.com/kata-containers/tests/pull/5392. IIUC this is an old limitation of runtime-rs (lack of vCPU resize) that was implemented for dragonball in https://github.com/kata-containers/kata-containers/pull/6289 . So maybe one can try to enable it on the dragonball job to see if it really works now?

vCPU resize should still not work with qemu-runtime-rs because CPU hot-plugging isn't implemented yet. Did I get it correctly @pmores ?

Anyway, I will write an accurate skip message to qemu-runtime-rs

Reply:
@wainersm vCPU resize is a matter of PR #9772 which is hopefully about to be merged. :-)